src/api/status/public/assets/js/github-stats.js

```diff
@@ -0,0 +1,68 @@
+const fetchGitHubApi = (owner, repo, path) => {
```

Comment:
Don't put it in `assets`, as these files are ignored by eslint and prettier. There should be an existing `public/js` directory.

Reply:
Thank you!